src/components/header/Header.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+import HeaderIcon from './HeaderIcon';
+import NavBar from './NavBar';
+
+const HeaderDiv = styled.div`
```

Comment:
Not sure if you're stuck because the tests are failing. The reason is this element. The tests expect a `header` instead of `div`.

Let me know if you need help